scripts/images/build_model_images.py

```
@@ -243,6 +282,17 @@ def resolve_output_flag(push: bool, load: bool, platform: str) -> str | None:
     return "--load"
 
 
+def image_reference_exists(image_reference: str) -> bool:
+    """Return whether a Docker image reference already exists in the registry."""
+    result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)